packages/ra-ui-materialui/src/list/Empty.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Inbox from '@material-ui/icons/Inbox';
+import { makeStyles } from '@material-ui/styles';
+import { useTranslate } from 'ra-core';
+import { CreateButton } from '../button';
+import inflection from 'inflection';
+
+const useStyles = makeStyles({
+    message: {
+        textAlign: 'center',
+        fontFamily: 'Roboto, sans-serif',
```

Comment:
Why do you need that? As it's in the layout, it should already have the right font.

Reply:
The hardcoded font is because I used `<NotFound />` as inspiration. Now it's not needed anymore.